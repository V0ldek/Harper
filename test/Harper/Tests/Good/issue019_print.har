main :: sideeffect -> ();
main = {
  eval print 42 ();
};