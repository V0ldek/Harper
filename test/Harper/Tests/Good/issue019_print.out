42

Execution ended with value:
()